src/sarif_converter.ts

```diff
@@ -3,14 +3,21 @@ import { Log, ReportingConfiguration, ReportingDescriptor, Result } from 'sarif'
 
 import { IAnalysisResult, IFileSuggestion } from './interfaces/analysis-result.interface';
 
+interface Fingerprint extends IFileSuggestion {
```

Comment:
It doesn't need to extend anything at all.

Reply:
Correct! removed it ... :)